### PR TITLE
Update Certificate information indicators, removing 404 url, adding new

### LINKED ISSUE #5192: 404 external link in kiali.io
make validate-site in kiali.io shows an external broken link: 

```
- ./public/docs/features/istio-component-status/index.html
  *  External link https://istio.io/latest/docs/tasks/security/cert-management/dns-cert/ failed: 404 No error

HTML-Proofer found 1 failure!
```

### PR DESCRIPTION
**Pull request by josunect**
Update Certificate information indicators, removing 404 url, addingew custom CA from k8s

Note: I've some doubts adding the new URL, as this is a feature in development. But, if this is in the Istio docs, I guess it can be added also in Kiali.io

Fixes [#5192 ](https://github.com/kiali/kiali/issues/5192)

### CODE REVIEW

File: content/en/docs/Features/istio-component-status.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The certificates shown depends on how Istio is configured. The following cases a
 
 * Using Istio CA certificates (default), the information shown is from a secret named *istio-ca-secret*.
 * Using [Plug in CA certificates](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/), the information shown is from a secret named *cacerts*.
-* Using [DNS certificates](https://istio.io/latest/docs/tasks/security/cert-management/dns-cert/), the information shown is from reading many secrets found in Istio configuration.
+* Using [Custom CA with Kubernetes CSR](https://istio.io/latest/docs/tasks/security/cert-management/custom-ca-k8s/), the information shown is from a custom CA integrated with the Kubernetes CSR API.
```
Comment (leandroberetta) [inline content/en/docs/Features/istio-component-status.md]: It seems that Istio removed support for DNS certificates in 1.14, they still are in 1.13: https://istio.io/v1.13/docs/tasks/security/cert-management/dns-cert/

Custom CA with Kubernetes CSR is not the same feature and we didn't implemented in Kiali yet, I guess we need to put the Istio's oldie link here and maybe mention someway it's not supported anymore in 1.14.

Comment (josunect) [reply content/en/docs/Features/istio-component-status.md]: Thanks @leandroberetta 
Adding change

**Review by leandroberetta — APPROVED** [review]
LGTM

**Comment by josunect** [comment]
Done in commit [221db18cb229ee64c759d1ae46176bfe4e57cfed](https://github.com/kiali/kiali.io/commit/221db18cb229ee64c759d1ae46176bfe4e57cfed)

**Pull request closed without merging**